src/js/cf-expandables.js

```diff
@@ -5,110 +5,185 @@
-        $this.removeClass('expandable__expanded');
-        $content.slideUp({
-          duration: duration,
-          easing: 'easeOutExpo'
-        });
-      };
+      }
 
-      this.init();
+      this.$target.attr( 'aria-controls', this.$content.attr( 'id' ) );
 
-    });
+      if ( $el.hasClass( this.expandedClass ) ) {
+        this.expand( 0 );
+      } else {
+        this.collapse( 0 );
+      }
+    },
 
-  };
+    initEvents: function initEvents() {
+      var toggle = $.proxy( _throttle, this, 450, this.toggle );
+      this.$el.on( 'click', '.expandable_target', toggle );
+    },
 
-  $.fn.expandable.calculateExpandDuration = function( height ) {
-    return $.fn.expandable.constrainValue( 450, 900, height * 4 );
-  };
+    toggle: function toggle( event ) {
+      event.preventDefault();
+      event.stopPropagation();
 
-  $.fn.expandable.calculateCollapseDuration = function( height ) {
-    return $.fn.expandable.constrainValue( 350, 900, height * 2 );
-  };
+      if ( this.isInAccordion ) {
+        this.$el.siblings( '.expandable' ).each( function( index, sibling ) {
+          sibling.collapse();
+        } );
+      }
+
+      if ( this.isExpanded ) {
+        this.collapse();
+      } else {
+        this.expand();
+      }
+    },
+
+    expand: function expand( duration ) {
+      var $content = this.$content;
+      this.$cueOpen.css( 'display', 'none' );
+      this.$cueClose.css( 'display', 'inline' );
+      this.$target.attr( 'aria-pressed', 'true' );
+      $content.attr( 'aria-expanded', 'true' );
+
+      if ( typeof duration === 'undefined' ) {
+        duration = $.fn.expandable.calculateExpandDuration( $content.height() );
+      }
+
+      this.$el.addClass( this.expandedClass );
+
+      $content.slideDown( {
+        duration: duration,
+        easing:   'easeOutExpo'
+      } );
+
+      this.isExpanded = true;
+    },
+
+    collapse: function collapse( duration ) {
+      var $content = this.$content;
+      this.$cueOpen.css( 'display', 'inline' );
+      this.$cueClose.css( 'display', 'none' );
+      this.$target.attr( 'aria-pressed', 'false' );
+      $content.attr( 'aria-expanded', 'false' );
+
+      if ( typeof duration === 'undefined' ) {
+        duration = $.fn.expandable.calculateExpandDuration( $content.height() );
+      }
+
+      this.$el.removeClass( this.expandedClass );
 
-  $.fn.expandable.constrainValue = function( min, max, duration ) {
-    if ( duration > max ) {
-        return max;
-    } else if ( duration < min ) {
-        return min;
-    } else {
-        return duration;
+      $content.slideUp( {
+        duration: duration,
+        easing:   'easeOutExpo'
+      } );
+
+      this.isExpanded = false;
     }
+
   };
 
-  // Auto init
-  $('.expandable').expandable();
 
-}(jQuery));
+  // Expandable Plugin declarations.
+
+  $.fn.expandable = function expandable( options ) {
+
+    return this.each( function() {
+      ( options || ( options = {} ) ).$el = $( this );
```

Comment:
If linting is set up for this, run this through ESLint. It may say the inside parentheses are not needed.